civihr_employee_portal/js/ta-documents-app.js

```diff
@@ -1,17 +1,17 @@
 /* globals angular */
 
 (function (angular) {
-  angular.module('taDocuments', ['civitasks.appDocuments', 'civitasks.directives'])
+  angular.module('taDocuments', ['tasks-assignments.documents', 'tasks-assignments.directives'])
     .config(function ($urlRouterProvider, $locationProvider) {
       $locationProvider.html5Mode(true); // This is required to remove # for the URL
       $urlRouterProvider.otherwise('/tasks-and-documents');
     })
     .controller('ModalController', ModalController);
 
   ModalController.$inject = ['$scope', '$rootScope', '$window', '$rootElement', '$log', '$uibModal',
-    'DocumentService', 'FileService', 'config', 'settings', 'DateFormat'];
+    'documentService', 'fileServiceTA', 'config', 'settings', 'DateFormat'];
```

Comment:
Why `fileServiceTA` and not `fileService`? Is there a specific file service for T&A?